### PR TITLE
Fix slider refresh when only one file in folder [stage-2]

### PR DESCRIPTION
**Pull request by stulees**
- Immediately destroying and recreating slider upon slider refresh (file change) and there's currently only one slide in total
- Integration tests added

### CODE REVIEW

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review, thanks. @Rise-Vision/delivery FYI

File: src/widget/slider.js [hunk 1]
```diff
@@ -262,8 +262,18 @@ RiseVision.Image.Slider = function( params ) {
...
     // Start preloading images right away.
     RiseVision.Common.Utilities.preloadImages( files );
-    newFiles = _.clone( files );
-    refreshSlider = true;
+
+    if ( currentFiles.length === 1 ) {
+      // Destroy and recreate the slider immediately if currently only 1 slide and there has been a change.
+      if ( $api ) {
+        destroySlider();
+        init( _.clone( files ) );
```
Comment (donnapep) [inline src/widget/slider.js]: @stulees Do we need to clone this? It's going to be done again [here](https://github.com/Rise-Vision/widget-image/blob/2a9d0d7a067a33453140305d3d63dda745de18fb/src/widget/slider.js#L198).

Comment (stulees) [inline src/widget/slider.js]: @donnapep We seem to be cloning it below in the `{ else }` block so I was keeping consistent. It does appear to be unnecessary though, I can remove.

**Comment by stulees** [comment]
@donnapep Change made, thanks.

**Comment by donnapep** [comment]
👍

**Pull request merged**